src/pages/home/ReportScreen.js

```diff
@@ -288,25 +297,14 @@ class ReportScreen extends React.Component {
                             }}
                         >
                             {(this.isReportReadyForDisplay() && !isLoadingInitialReportActions) && (
-                                <>
-                                    <ReportActionsView
-                                        reportActions={this.props.reportActions}
-                                        report={this.props.report}
-                                        session={this.props.session}
-                                        isComposerFullSize={this.props.isComposerFullSize}
-                                        isDrawerOpen={this.props.isDrawerOpen}
-                                        parentViewHeight={this.state.skeletonViewContainerHeight}
-                                    />
-                                    <ReportFooter
-                                        errors={addWorkspaceRoomOrChatErrors}
-                                        pendingAction={addWorkspaceRoomOrChatPendingAction}
-                                        isOffline={this.props.network.isOffline}
-                                        reportActions={this.props.reportActions}
-                                        report={this.props.report}
-                                        isComposerFullSize={this.props.isComposerFullSize}
-                                        onSubmitComment={this.onSubmitComment}
-                                    />
-                                </>
```

Comment:
I'm not following why this is being moved from here. Can we just leave it and make sure to show a default footer like we are doing in the `ReportActionsSkeletonView`. I had a previous suggestion to move the placeholder footer into that component. Did we try that or not?

Reply:
That conversation makes no sense to me can you please summarize in one or two sentences why it does not work.

Reply:
@0xmiroslav can you please provide a brief summary for us so we can move this along? One or two sentences would be perfect. I can better assist if you provide a clear explanation vs. links to videos or other conversations.